### DISCUSSION TITLE
Pass through arg/parameter

### DISCUSSION THREAD

**Discussion by benito2313** [Miksus/rocketry#198] [Q&A] [answered]
Hi,
I would like to pass through a argument/parameter when a job is executing another py file.

Setup now is the file that i want to access/run has the parameters set.
def run(arg1)

When i run the rocketry and pass the values in app.task(func_name=run, path=script.py, parameters={arg1="y"})

But it isnt passing the values through.
What am i doing wrong?

Kind regards,

Benito2313

**Comment by benito2313** [comment, marked as answer]
Got it to work.

**Reply by kwatog** [reply]
how did you get it to work? I'm doing the same thing but I can't receive the param.

**Reply by kwatog** [reply]
nevermind. the code the worked for me is as below
```
app_rocketry.task(
            func_name="run",
            execution="main",
            start_cond="minutely",
            path="app/func/operator.py",
            parameters={"arg1":"y"}
            )
```
then in my run function..

```
def run(**kwargs):
    blah..blah...
```
